src/components/ProductTours/AdminOnboardingTours/data/useFetchCompletedOnboardingFlows.ts

```diff
@@ -0,0 +1,29 @@
+import { useQuery } from '@tanstack/react-query';
+
+import LmsApiService from '../../../../data/services/LmsApiService';
+import { adminOnboardingQueryKeys } from '../constants';
+import { camelCaseDict } from '../../../../utils';
+
+export type CompletedTourFlows = {
+  completedTourFlows: [string];
+};
+
+/**
+ * Fetching the enterprise customer admin's profile to see their completed tour flows
+ *
+ * @param {adminUuid} queryKey The admin's associated uuid
+ * @returns {CompletedTourFlows} Array of completed tour flow uuids
+ */
+
+export const getFetchCompletedOnboardingFlows = async ({ adminUuid }) : Promise<CompletedTourFlows> => {
```

Comment:
This is currently returning a list of tour flow uuids, not the full `CompletedTourFlows` object.
```suggestion
export const getFetchCompletedOnboardingFlows = async ({ adminUuid }) : Promise<string[]> => {
```